bootstrap/kubeadm/api/v1alpha2/zz_generated.conversion.go

```diff
@@ -289,14 +289,10 @@ func autoConvert_v1alpha3_KubeadmConfigSpec_To_v1alpha2_KubeadmConfigSpec(in *v1
 	out.Users = *(*[]User)(unsafe.Pointer(&in.Users))
 	out.NTP = (*NTP)(unsafe.Pointer(in.NTP))
 	out.Format = Format(in.Format)
+	// WARNING: in.Verbosity requires manual conversion: does not exist in peer-type
```

Comment:
We need to add this to `conversion.go` and restore the value, like we do for other fields that don't exist in v1alpha2

Reply:
do you have an example of conversion for KubeadmConfig fields?

Reply:
example: https://github.com/kubernetes-sigs/cluster-api/blob/master/bootstrap/kubeadm/api/v1alpha2/conversion.go#L39

Reply:
since Verbosity is optional, how do we translate that in conversion? Wouldn't it just be empty by default?

Reply:
So I added the conversion above in conversion.go but looks like that wasn't enough to make the warning go away

Reply:
Nevermind I think that's expected